modules/ROOT/pages/dw-core.adoc

```diff
@@ -26,12 +26,12 @@ attributes in the input object.
 returns only matching objects, filtering out the rest from the output.
 | xref:dw-core-functions-find.adoc[find] | Returns indices of an input that match a specified value.
 | xref:dw-core-functions-flatmap.adoc[flatMap] | Iterates over each item in an array and flattens the results.
-| xref:dw-core-functions-flatten.adoc[flatten] | Turns a set of subarrays (such as [ `[1,2,3], [4,5,[6]], [], [null] ]`) into a single, flattened array (such as `[ 1, 2, 3, 4, 5, [6], null ]`).
+| xref:dw-core-functions-flatten.adoc[flatten] | Helper function that enables `flatten` to work with a `null` value.
 | xref:dw-core-functions-floor.adoc[floor] | Rounds a number down to the nearest whole number.
 | xref:dw-core-functions-groupby.adoc[groupBy] | Returns an object that groups items from an array based on specified
 criteria, such as an expression or matching selector.
-| xref:dw-core-functions-isblank.adoc[isBlank] | Returns `true` if the given string is empty or completely composed of whitespace,
-`false` if not.
+| xref:dw-core-functions-indexof.adoc[indexOf] | Returns the index of the *first* occurrence of the specified element in this Array, or -1 if this list does not contain the element.
```

Comment:
TODO: Fix description of indexOf so that *first* -> _first_, and copyedit the whole description.